qiskit/visualization/qcstyle.py

```diff
@@ -30,8 +30,8 @@ class DefaultStyle:
     `latex` output. The options available in the style dict are defined below:
 
     name (str): the name of the style. The name can be set to ``iqx``,
-        ``bw``, ``default``, or the name of a user-created json file. This
-        overrides the setting in the user config file (usually
+        ``textbook``, ``bw``, ``default``, or the name of a user-created json
+        file. This overrides the setting in the user config file (usually
```

Comment:
Since you're updating the doc string here, it would probably be a good idea to add "iqx" and "iqx-dark" to the list of styles we supply.